To regenerate the patch, get the full wxWidgets tarball, copy build/bakefiles
and the various *.bkl files into the wxGTK tree and run
	./bakefile_gen -f autoconf
in build/bakefiles. manual-bakefile.diff contains the necessary patch for
the bakefile ruleset to use libtool.

diff -uNr wxWidgets-2.8.10.orig/build/bakefiles/regex.bkl wxWidgets-2.8.10/build/bakefiles/regex.bkl
--- wxWidgets-2.8.10.orig/build/bakefiles/regex.bkl	2009-03-06 13:10:54.000000000 +0100
+++ wxWidgets-2.8.10/build/bakefiles/regex.bkl	2009-03-22 20:09:05.000000000 +0100
@@ -27,16 +27,13 @@
         <if cond="wxUSE_REGEX=='builtin'">$(TOP_SRCDIR)src/regex</if>
     </set>
 
-    <lib id="wxregex" template="msvc_setup_h,3rdparty_lib"
+    <dll id="wxregex" template="msvc_setup_h,anylib"
          cond="wxUSE_REGEX=='builtin' and BUILDING_LIB=='1'">
-        <libname cond="FORMAT=='autoconf'">
-            $(id)$(WXUNICODEFLAG)$(WXDEBUGFLAG)$(WX_LIB_FLAVOUR)-$(WX_RELEASE)$(HOST_SUFFIX)
-        </libname>
-        <libname cond="FORMAT!='autoconf'">
-            $(id)$(WXUNICODEFLAG)$(WXDEBUGFLAG)$(HOST_SUFFIX)
-        </libname>
-        <include cond="FORMAT!='autoconf'">$(TOP_SRCDIR)include</include>
-        <include cond="FORMAT!='autoconf'">$(SETUPHDIR)</include>
+        <dllname>
+            $(id)$(WXUNICODEFLAG)$(WXDEBUGFLAG)$(WX_LIB_FLAVOUR)-$(WX_RELEASE)
+	</dllname>
+        <version>$(WX_VERSION)</version>
+        <so_version>$(WXSOVERSION)</so_version>
         <define>__WX$(TOOLKIT)__</define>
         <define>$(WXUNIV_DEFINE)</define>
         <define>$(UNICODE_DEFINE)</define>
@@ -49,6 +46,6 @@
             src/regex/regerror.c
             src/regex/regfree.c
         </sources>
-    </lib>
+    </dll>
 
 </makefile>
diff -uNr wxWidgets-2.8.10.orig/build/bakefiles/wx.bkl wxWidgets-2.8.10/build/bakefiles/wx.bkl
--- wxWidgets-2.8.10.orig/build/bakefiles/wx.bkl	2009-03-06 13:10:54.000000000 +0100
+++ wxWidgets-2.8.10/build/bakefiles/wx.bkl	2009-03-22 20:09:05.000000000 +0100
@@ -96,7 +96,7 @@
             <command>
                 $(INSTALL_DIR) $(DESTDIR)$(BINDIR)
                 $(INSTALL_DIR) $(DESTDIR)$(LIBDIR)/wx/config
-                $(INSTALL_PROGRAM) lib/wx/config/$(TOOLCHAIN_FULLNAME) $(DESTDIR)$(LIBDIR)/wx/config
+                ${BSD_INSTALL_SCRIPT} lib/wx/config/$(TOOLCHAIN_FULLNAME) $(DESTDIR)$(LIBDIR)/wx/config
                 (cd $(DESTDIR)$(BINDIR) &amp;&amp; rm -f wx-config &amp;&amp; $(LN_S) $(LIBDIR)/wx/config/$(TOOLCHAIN_FULLNAME) wx-config)
             </command>
         </action>
diff -uNr wxWidgets-2.8.10.orig/utils/wxrc/wxrc.bkl wxWidgets-2.8.10/utils/wxrc/wxrc.bkl
--- wxWidgets-2.8.10.orig/utils/wxrc/wxrc.bkl	2009-03-06 13:11:15.000000000 +0100
+++ wxWidgets-2.8.10/utils/wxrc/wxrc.bkl	2009-03-22 20:10:17.000000000 +0100
@@ -21,7 +21,7 @@
         <modify-target target="install_wxrc">
             <command>
 	        rm -f $(DESTDIR)$(BINDIR)/wxrc$(EXEEXT) $(DESTDIR)$(BINDIR)/$(versioned_name)
-		$(INSTALL_PROGRAM) wxrc$(EXEEXT) $(DESTDIR)$(BINDIR)
+		${LIBTOOL} --mode=install $(INSTALL_PROGRAM) wxrc$(EXEEXT) $(DESTDIR)$(BINDIR)
                 mv -f $(DESTDIR)$(BINDIR)/wxrc$(EXEEXT) $(DESTDIR)$(BINDIR)/$(versioned_name)
                 (cd $(DESTDIR)$(BINDIR) &amp;&amp; $(LN_S) $(versioned_name) wxrc$(EXEEXT))
             </command>
